cypress/integration/pages/topicPage/tests.js

```diff
@@ -0,0 +1,116 @@
+import envConfig from '../../../support/config/envs';
+
+export default ({ service, pageType }) => {
+  describe(`Tests for ${service} ${pageType}`, () => {
+    let topicId;
```

Comment:
this variable could be deleted and be a const on line 11 because it never changes